app/src/main/java/org/apache/fineract/ui/online/login/LoginActivity.java

```diff
@@ -118,4 +126,13 @@ protected void onDestroy() {
         hideMifosProgressDialog();
         loginPresenter.detachView();
     }
+
+    private void hideSoftKeyboard() {
```

Comment:
Good work! can you implement this method in `Utils` class, please? So we can use it in any place we want it.

Reply:
Sure

Reply:
Please make the changes asap if you can, we need to merge the PRs. Thanks : )